app/views/adoptions/_request.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if adoption %>
+  <h3 class="t-list__heading"> You have requested adoption for <%= @rubygem.name %>:</h3>
+  <%= render partial: "adoptions/show", locals: { adoption: adoption } %>
+<% elsif signed_in? %>
+  <div class="gem_intro">
+    <h3 class="t-list__heading">Request adoption :</h3>
```

Comment:
```suggestion
    <h3 class="t-list__heading">Request adoption:</h3>
```

Reply:
are you sure there will be a space before colon?
https://english.stackexchange.com/questions/67394/spaces-around-a-colon